react/tic-tac-toe/src/app/screens/App/components/Login/components/Field/styles.module.scss

```diff
@@ -0,0 +1,62 @@
+@import '../../../../../../../assets/variables';
+@import '../../../../../../../assets/colors';
+
+.fieldContainer {
+  margin-top: 1.5rem;
+
+  &.invalid {
+    .fieldInput {
+      border-color: $red;
+    }
+    .fieldLabel {
+      color: $red
+    }
+  }
+
+  &.valid {
+    .fieldInput {
+      border-color: $blue;
+    }
+    .fieldLabel {
+      color: $blue
+    }
+  }
+}
+
+.fieldError {
+  color: $red;
+  display: block;
+  font-size: $text-small;
+  transform: translateY(-1rem);
+}
+
+.fieldLabel {
+  display: block;
+  font-family: 'Roboto', sans-serif;
+  font-size: 1rem;
+  transform: translate(-4rem, -2.5rem) scale(0.7);
+  transition: all 0.3s;
```

Comment:
Try not to apply transitions to all properties since it may result in a performance bottleneck when dealing with multiple transitions at the same time. If you're just changing the `transform` property, you can explicitly declare it. 🙂